pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -831,7 +831,7 @@ func (k *Bootstrapper) UpdateNode(cfg config.ClusterConfig, n config.Node, r cru
 		return errors.Wrap(err, "host alias")
 	}
 
-	return sm.Start("kubelet")
```

Comment:
Can you confirm that Kubernetes v1.13.x starts up with this change?

Reply:
confirmed, takes 35s on my machine